geth/jail/execution_policy.go

```diff
@@ -142,6 +142,72 @@ func (ep *ExecutionPolicy) executeLocalSendTransaction(req common.RPCCall, call
 	return resp, nil
 }
 
+// ExecuteSHH defines a function which handles the processing of `shh_*` transaction methods
+// rpc request to the internal node server.
+func (ep *ExecutionPolicy) ExecuteSHH(req common.RPCCall, call otto.FunctionCall) (*otto.Object, error) {
```

Comment:
This looks like a copy paste of `ExecuteOtherTransaction()`. Maybe we could somehow combine them?